apps.yaml

```diff
@@ -263,3 +263,18 @@ tomato:
         - technology-lab
         - stakeholder
         - stakeholder-aurora
+aurora-app:
+    metadata:
+        title: Aurora-app
+        description: |
+            AiiDAlab app for the Autonomous robotic battery innovation platform (Aurora) BIG-MAP Stakeholder initiative
+        authors: Loris Ercole
+        documentation_url: https://github.com/epfl-theos/aurora-app/blob/main/README.md
+        stage: registered
+    git_url: https://github.com/epfl-theos/aurora-app
```

Comment:
@lorisercole Do you want to consider to maintain this app in the https://github.com/aiidalab organization? In this case, we usually follow the convention `aiidalab-<app-name>` for the repository name, e.g., `aiidalab-aurora`. Please let me know.

Reply:
Oh, I guess so. @giovannipizzi what do you think?

Reply:
Ok, I implemented your suggestion. Also the github repo has been transferred to the new name.

Reply:
Thanks! Do you want to move it to the 'aiidalab' GitHub organization as well?

Reply:
Oh, I have asked @giovannipizzi. I'll let you asap!

Reply:
In the end we decided to keep it under 'epfl-theos' for now

Reply:
Ok, thanks for letting me know.